notes/SDLC.md

```diff
@@ -0,0 +1,54 @@
+# SDLC 
+## SLDC models: Waterfall vs Agile
+### Waterfull model  
+* Phases
+    1. Requirements
+    2. Design 
+    3. implementation
+    4. Verification
+    5. Maintenance
+* Projects progress one step at a time in a linear fashion until a project reaches maintenance
+	* "one should move to a phase only when its preceding phase is reviewed and verified" 
+* Pros 
+	* Model places significant focus on requirements and documentation
+	* Designed to catch issues as early as possible (see cost of a bug)
```

Comment:
make "cost of bug" a link to the relevant resource

Reply:
fixed